code/models/SetOfSet.py

```diff
@@ -55,8 +57,8 @@ def __init__(self, conf):
         self.m_net = get_linear_layers([num_feats] * 2 + [m_d_out], final_layer=True, batchnorm=False)
         self.n_net = get_linear_layers([num_feats] * 2 + [n_d_out], final_layer=True, batchnorm=False)
 
-    def forward(self, data):
-        x = data.x  # x is [m,n,d] sparse matrix
+    def forward(self, data: SceneData):
+        x: SparseMat = data.x  # x is [m,n,d] sparse matrix
```

Comment:
Let's remove this

Reply:
It's helpful for pycharm suggestions 
Or I can do it and mention a commit 57f41c0